docs/source/guides/performance.rst

```diff
@@ -35,9 +61,54 @@ We can also set chunk size to be a percentage of total rows::
                                            target_entity="customers",
                                            chunk_size=.05)
 
+Using persistent cluster
```

Comment:
i think this header can go one level up since it is related to "Simple Parallel Feature Computation"